models/quality_measures/intermediate/nqf0059_diabetes_hemoglobin_a1c/quality_measures__int_nqf0059_numerator.sql

```diff
@@ -47,6 +47,7 @@ with denominator as (
       labs.patient_id
     , labs.result
     , coalesce(collection_date,result_date) as evidence_date
+    , result as evidence_value
```

Comment:
It looks like we're including "result" twice. Can we just alias `labs.result` as "evidence_value"?